content/docs/compute/custom-image.md

```diff
@@ -0,0 +1,184 @@
+
+```console
+$ civo diskimage show cust-my-ubuntu-image
+                ID : 8a76ec9c-ef94-4d21-90a5-b48fc6d4fee2
+              Name : cust-my-ubuntu-image
+      Distribution : ubuntu
+           Version : 22.04
+                OS : linux
+             State : available
+         Image URL : https://objectstore.staging.civo.com/disk-images/my-ubuntu-image/disk.raw?X-Amz-Algorithm=AWS4-HMAC-SHA256&SignedHeaders=host&x-id=PutObject&X-Amz-Signature=8a7d32f...
+          Logo URL : https://objectstore.staging.civo.com/disk-images/my-ubuntu-image/logo.svg
+       Image Size : 2.5 GB
+```
+
+</TabItem>
+</Tabs>
+
+### Understanding disk image states
+
+The disk image states include:
+
+- `upload_pending`: Initial state after creation, awaiting upload
+- `upload_completed`: Upload finished, MD5 checksum verified
+- `pending`: Image is being processed for distribution
+- `available`: Image is ready to use for creating instances
+- `upload_expired`: Upload not completed within 15 minutes
+- `error`: Issue with upload or processing
+
+:::note
+Only images in the **available** state can be used to create instances.
+:::
+
+## Listing custom disk images
+
+<Tabs groupId="list-diskimage">
+<TabItem value="cli" label="Civo CLI">
+
+To see all your custom disk images:
+
+```console
+$ civo diskimage list --local
++--------------------------------------+----------------------+--------------+---------+-----------+
+| ID                                   | Name                 | Distribution | Version | Status    |
++--------------------------------------+----------------------+--------------+---------+-----------+
+| 8a76ec9c-ef94-4d21-90a5-b48fc6d4fee2 | cust-my-ubuntu-image | ubuntu       | 22.04   | available |
++--------------------------------------+----------------------+--------------+---------+-----------+
+```
+
+The `--local` flag filters the list to show only your custom uploaded disk images, excluding system images.
+
+</TabItem>
+</Tabs>
+
+## Deleting custom disk images
+
+<Tabs groupId="delete-diskimage">
+<TabItem value="cli" label="Civo CLI">
+
+To delete a custom disk image:
+
+```console
+$ civo diskimage delete cust-my-ubuntu-image
+The disk image (cust-my-ubuntu-image) has been deleted
+```
+
+:::note
+You cannot delete a custom disk image that is currently in use by any instances.
+:::
+
+</TabItem>
+</Tabs>
+
+## Creating instances using custom disk images
+
+<Tabs groupId="use-diskimage">
+<TabItem value="cli" label="Civo CLI">
+
+Once your custom disk image is in the **available** state, you can use it to create new instances:
+
+```console
+$ civo instance create --hostname custom-instance \
+                     --size g3.small \
+                     --diskimage cust-my-ubuntu-image
+Created instance custom-instance with ID 58a9b7f2-5d12-46ab-9e97-b3d24c32cc2a
+```
+
+</TabItem>
+</Tabs>
+
+## Troubleshooting
+
+### Upload expired or error state
+
+If your upload expired or error state:
```

Comment:
This should probably be "If your upload window expired, or the image is in an error state:"